x-pack/qa/rolling-upgrade/build.gradle

```diff
@@ -137,7 +137,10 @@ for (Version bwcVersion : bwcVersions.wireCompatible) {
     if (bwcVersion.before('7.4.0')) {
       toBlackList.addAll([
         'mixed_cluster/80_transform_jobs_crud/Test GET, start, and stop old cluster batch transforms',
-        'mixed_cluster/80_transform_jobs_crud/Test GET, stop, start, old continuous transforms'
+        'mixed_cluster/80_transform_jobs_crud/Test GET, stop, start, old continuous transforms',
+        //Cannot determine if we will throw a warning on get as we might hit an old node
+        //Cannot determine if we will have a warning on get _stats as we might hit an old node
+        'mixed_cluster/40_ml_datafeed_crud/Test old cluster datafeed with aggs'
```

Comment:
This is frustrating, but I cannot think of a way to "sometimes" expected a warning header. We just don't know if the call would hit an old node, a new node, or is serialized between the two.

Reply:
I agree.  This is not great.

Because the tests are being so picky about non-fatal warnings we cannot test the functionality at all which means we might ship a fatal error.

It seems that the framework needs a "tolerate this warning but don't require it" option.  Then we could test that doing a minor version upgrade of a cluster that is using deprecated functionality works.